frontends/p4/typeChecking/typeChecker.cpp

```diff
@@ -3300,15 +3310,25 @@ const IR::Node* TypeInference::postorder(IR::Property* prop) {
                 return prop;
             }
             auto at = type->to<IR::Type_Action>();
-            if (at->parameters->size() != 0)
+            if (at->parameters->size() != 0) {
                 typeError("Action for %1% has some unbound arguments", prop->value);
+                return prop;
```

Comment:
Would be nice to identify those arguments so that folks who do not debug the compiler can help themselves.

Reply:
Actually I tried to create a test case that triggers this error, but I think other type checking cases will report an error first, so this never seems to happen. I think that this is defensive programming, so I will leave it in. I will commit a version with a better error message, but I could not create a test to exercise it.